seirmo/figures.py

```diff
@@ -0,0 +1,121 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import plotly.graph_objects as go
+
+import warnings
+
+
+class IncidenceNumberPlot(object):
+    """
+
+    A figure class that illustrates the incidence number time
+    series as a bar plot over time.
+
+    """
+    def __init__(self):
+        super(IncidenceNumberPlot, self).__init__()
+
+        # Instantiate an empty plotly figure
+        self._fig = go.Figure()
+
+    def add_data(self, data, time_key='Time',
+                 inc_key='Incidence Number'):
+        """
```

Comment:
The way you would usually break the arguments here is to say
```
def add_data(
        self, data, time_key='Time', inc_key='Incidence Number'):
    """
    docs
    """
```
I.e. two tabs (first tab for body of the function, second tab to continue the arguments).